docs/content/common-subs/required-packages.md

```diff
@@ -60,7 +60,141 @@
         choco install yq -y
         ```
         ``` title="kubectl - https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/ (version range expected: 1.23-1.25)"
-        curl.exe -LO "https://dl.k8s.io/release/v1.27.2/bin/windows/amd64/kubectl.exe"
+        curl.exe -LO "https://dl.k8s.io/release/v1.27.2/bin/windows/amd64/kubectl.exe"    
+        ```  
+    === "WSL with Ubuntu"  
+        ```
+        ```
+        # How to install pre-requisites for a Windows Subsystem for Linux (WSL) envronment using an Ubuntu 22.04.01 distribution
+
+        Tested on a Intel(R) Core(TM) i7-9850H CPU @ 2.60GHz 2.59 GHz with 32GB RAM, a 64-bit operating system, x64-based processor
+        Using Windows 11 Enterprise
+
+        # 1. NB: If you're using a VPN, turn it off
+
+        # 2. Install Ubuntu into WSL
+
```

Comment:
you actually need a step 2.0

         ## 2.0 If not already done, activate Windows Subsystem for linux with the basic command
         wsl --install
         In my case this installed Ubuntu 22.04.33